src/Ilios/CoreBundle/Entity/Manager/PermissionManager.php

```diff
@@ -125,6 +125,58 @@ public function userHasReadPermissionToSchool(UserInterface $user, SchoolInterfa
     /**
      * {@inheritdoc}
      */
+    public function userHasReadPermissionToSchools(UserInterface $user, ArrayCollection $schools)
+    {
+        $criteria = [
+            'tableName'     => 'schools',
+            self::CAN_READ  => true,
+            'user'          => $user,
+        ];
+
+        $schoolsWithReadPermission = $this->findPermissionBy($criteria);
+        
+        if ($schoolsWithReadPermission->count() === 0) {
+            return false;
+        }
+        
+        foreach ($schools as $school) {
```

Comment:
Good change.  I was trying to find a forAny method on the doctrine array collection and then forgot to fall back to PHP.